### DISCUSSION TITLE
Inactive Meetings Not Showing Up when day is selected

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#965] [Q&A] [answered]
I changed status to inactive for a meeting that is going on hiatus for a couple of months. 

In TSML-UI the meeting does not appear on the list if I select Any Type and the day it met. If I let it default to Any Type and Any Day and then scroll down until I hit Monday (which was the day it met), it will be on this list. If I select Inactive, but not limit the day, it will show up. The issue is when selecting the day.

![Monday-Inactive-Meetings](https://user-images.githubusercontent.com/62247421/199268112-f6af7a92-26e6-43a8-8f6a-49726698d4b8.png)

When I remove Monday in the above example, the inactive meeting is displayed.
![Inactive-Meetings](https://user-images.githubusercontent.com/62247421/199268564-8b6f4466-6a5f-4319-88e0-aa3eae03b569.png)

**Comment by joshreisner** [comment, marked as answer]
Hi Viv, that's right, inactive meetings are specifically excluded from dropdown filtering. This is so that if, say, there is only one LGBTQ meeting and it is inactive, then that type does not show in the Types menu with a [1] next to it, disappointing users.

The exception, of course, is the `Inactive` type.

**Reply by Grateful-Viv** [reply]
Thanks for the clarification, @joshreisner

**Reply by Grateful-Viv** [reply]
We had two more meetings change from active to inactive. I think they're going to come back so I did not put them in draft mode. 

But these meetings do not show up on the meeting list grid. It does show up if I select inactive from the dropdown. But I don't think people are going to know to look there.  I know the Meeting Guide App will display it with that ugly gray color with the X through it. 

I am concerned if someone is expecting to go to the meeting and didn't know it was set as inactive because it's not displaying on the list. 

You can see from this image below the first meeting showing up is at 8AM
 ![Saturday-Meetings](https://user-images.githubusercontent.com/62247421/204315558-9c5bac3a-b4b8-49d8-9126-69f3add7d831.png)

Yet only if I select inactive for any day will these results be returned.
![Inactive-Meetings (Wake Up)](https://user-images.githubusercontent.com/62247421/204319058-e015f28d-8de6-49c0-a98c-a8a0773a8032.png)

Can this be changed? I would like the inactive meetings to show up on the list or at least be at the top of the dropdown where it says Any Type followed by Active (with a count). Maybe Inactive (with a count) could display under it?
![Friday-Meetings](https://user-images.githubusercontent.com/62247421/204321690-c2fd39a8-a14c-4586-a767-061d588db593.png)

**Reply by joshreisner** [reply]
hi @Grateful-Viv inactive meetings do show up in the default and most common "any day" view; it is not showing up for you in that image only because Saturday is selected.

i agree that users are not likely to select inactive, but do think the most common view is with nothing selected.

as i said above, i think it would be a bad user experience to count these meetings among the other available filters. my main concern is that there could be only inactive meetings for a given type, so a user picks lgbtq, for example, and is let down. this would seem to me a "broken" ux.

**Reply by Grateful-Viv** [reply]
So you’re saying if I didn’t select a particular day or a particular type of meeting (e.g. LGBTQ) that it would show up on the list? Since today is only Monday, I have to scroll down without any filtering until I got to Friday or Saturday and it would be there. That makes sense. Personally I wish we didn’t have to have inactive meetings, because we’re back to that business about the location being temporally closed, which is not necessarily so. It’s just the meeting is closed. Thanks @joshreisner.
